core/src/accounts_hash_verifier.rs

```diff
@@ -155,17 +179,53 @@ impl AccountsHashVerifier {
             }
         }
 
+        cluster_info.push_accounts_hashes(hashes.clone());
+    }
+
+    fn handle_snapshot(
+        accounts_package: AccountsPackage,
+        pending_snapshot_package: Option<&PendingSnapshotPackage>,
+        snapshot_config: Option<&SnapshotConfig>,
+    ) {
```

Comment:
This function has the code for now doing an Incremental Snapshot.

A nice thing that came out of the changes, is now we don't unconditionally make a `SnapshotPackage`. Instead, only when it's time. This should save a bit of time copying/moving data around. And in the case of an Incremental Snapshot, this is where the snapshot storages filtering takes place, and not anywhere else.